<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lays</author>
 <class>BrPrint3D</class>
 <widget class="QMainWindow" name="BrPrint3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>BrPrint3D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Icons/Icons/logo</normaloff>:/Icons/Icons/logo</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_hide">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Configuration - Show</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>980</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bt_import">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/Icons/Icons/bt-import</normaloff>:/Icons/Icons/bt-import</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_open">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/Icons/Icons/bt-file</normaloff>:/Icons/Icons/bt-file</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_connect">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/Icons/Icons/bt-connect</normaloff>
         <normalon>:/Icons/Icons/bt-connect-onclick</normalon>:/Icons/Icons/bt-connect</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/Icons/Icons/brlogo.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bt_play">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/Icons/Icons/bt-play</normaloff>
         <disabledon>:/Icons/Icons/bt-play-onclick</disabledon>:/Icons/Icons/bt-play</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_stop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/Icons/Icons/bt-stop</normaloff>:/Icons/Icons/bt-stop</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_pause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/Icons/Icons/bt-pause</normaloff>
         <normalon>:/Icons/Icons/bt-pause-onclick</normalon>:/Icons/Icons/bt-pause</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_emergency">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/Icons/Icons/bt-emergency.png</normaloff>:/Icons/Icons/bt-emergency.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>145</y>
      <width>1016</width>
      <height>550</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="PrinterSettingsWidget" name="PrinterSettings" native="true">
     <property name="minimumSize">
      <size>
       <width>390</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>390</width>
       <height>550</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <layout class="QHBoxLayout" name="vtkConteiner"/>
    </widget>
    <widget class="QWidget" name="ManualControl" native="true">
     <property name="minimumSize">
      <size>
       <width>410</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>410</width>
       <height>550</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFechar"/>
   </widget>
   <widget class="QMenu" name="menuLinguagem">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionPortuguese"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAviso_Legal"/>
    <addaction name="actionDocumenta_o"/>
    <addaction name="actionAjuda"/>
    <addaction name="actionSobre_o_BrPrint3D"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuLinguagem"/>
   <addaction name="menuSobre"/>
  </widget>
  <action name="actionEmergency">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Icons/Icons/bt-emergency.png</normaloff>:/Icons/Icons/bt-emergency.png</iconset>
   </property>
   <property name="text">
    <string>Parada de Emergência</string>
   </property>
   <property name="toolTip">
    <string>Clique aqui para interromper a impressão em caso de emergência.</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English - EUA</string>
   </property>
  </action>
  <action name="actionPortuguese">
   <property name="text">
    <string>Portuguese - Brazil</string>
   </property>
  </action>
  <action name="actionAviso_Legal">
   <property name="text">
    <string>Legal Warning</string>
   </property>
  </action>
  <action name="actionFechar">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImportG">
   <property name="text">
    <string>Importar GCode</string>
   </property>
  </action>
  <action name="actionCores_Visualiza_o">
   <property name="text">
    <string>Cores Visualização</string>
   </property>
  </action>
  <action name="actionTema">
   <property name="text">
    <string>Tema</string>
   </property>
  </action>
  <action name="actionDocumenta_o">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAjuda">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Icons/Icons/bt-play</normaloff>
     <disabledoff>:/Icons/Icons/bt-play-onclick</disabledoff>
     <disabledon>:/Icons/Icons/bt-play-onclick</disabledon>:/Icons/Icons/bt-play</iconset>
   </property>
   <property name="text">
    <string>Executar</string>
   </property>
   <property name="toolTip">
    <string>Clique aqui para iniciar a impressão.</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Icons/Icons/bt-import</normaloff>:/Icons/Icons/bt-import</iconset>
   </property>
   <property name="text">
    <string>Importar</string>
   </property>
   <property name="toolTip">
    <string>Clique aqui para importar um GCode</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Icons/Icons/bt-stop</normaloff>
     <activeoff>:/new/Images/bt-stop</activeoff>
     <activeon>:/Icons/Icons/bt-stop-onclick</activeon>:/Icons/Icons/bt-stop</iconset>
   </property>
   <property name="text">
    <string>Parar</string>
   </property>
   <property name="toolTip">
    <string>Clique aqui para cancelar a impressão.</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Icons/Icons/bt-connect</normaloff>
     <activeon>:/Icons/Icons/bt-connect-onclick</activeon>:/Icons/Icons/bt-connect</iconset>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
   <property name="toolTip">
    <string>Clique aqui para conectar a impressora.</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Icons/Icons/bt-pause</normaloff>
     <activeon>:/Icons/Icons/bt-pause-onclick</activeon>:/Icons/Icons/bt-pause</iconset>
   </property>
   <property name="text">
    <string>Pausar</string>
   </property>
   <property name="toolTip">
    <string>Clique aqui para pausar a impressão.</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Icons/Icons/bt-file</normaloff>:/Icons/Icons/bt-file</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="toolTip">
    <string>Clique aqui para abrir um arquivo.</string>
   </property>
  </action>
  <action name="actionSobre_o_BrPrint3D">
   <property name="text">
    <string>About Br-Print3D</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrinterSettingsWidget</class>
   <extends>QWidget</extends>
   <header>PrinterSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bt_hide</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
